OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix reducer phase2

# Description
This is lint fix for all the file under src/reducers/ except: actionltemsReducer.js, allProjectsReducer.js, allTasksReducer.js, allTeamsReducer.js, allTimeEntriesReducer.js from src/reducers file

I haven't changed the export to a default export because I've found that even if I make the change and remove the curly braces in the index.js file, something still goes wrong.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user

(Please be sure you record a video)
5. **Verify that the reducer is working normally.**



(No need to test below)
~~6. Delete .eslintignore file in the root: `rm .eslintignore`
7. `npx eslint all the file under src/reducers/ except: actionltemsReducer.js, allProjectsReducer.js, allTasksReducer.js, allTeamsReducer.js, allTimeEntriesReducer.js from src/reducers file`
11. `git restore .`~~
